src/main/java/JanggiApplication.java

```diff
@@ -0,0 +1,20 @@
+import domain.Game;
+import view.InputView;
+import view.OutputView;
+
+public class JanggiApplication {
+
+    public static void main(String[] args) {
+        final InputView inputView = new InputView();
+        final OutputView outputView = new OutputView();
+        final Game game = new Game(inputView, outputView);
+
+        while (!game.isEnd()) {
```

Comment:
사소할 수 있지만 부정문보다는 긍정문을 쓰는 것이 조금 더 가독성이 좋다고 생각해요. 😃
지금의 경우에도 isEnd의 부정보다는 isPlaying 같이 현재 게임이 진행중이라면 while 문 안의 로직을 수행한다고 코드가 자연스레 읽힐 것 같아요. 

```suggestion
        while (game.isPlaing()) {
```